examples/tests/test_examples.py

```diff
@@ -148,7 +148,7 @@ def test_examples_compare(filename, pytestconfig, force_offscreen, mock_time):
     stored_img = iio.imread(screenshot_path)
 
     # assert similarity
-    atol = 2
+    atol = 1
```

Comment:
wow. i somewhat don't believe this.....

i was seeing variations in the middle of the `k` .... i guess your changes made it more stable for "small font" which I think we were always operating in that limit.